web/packages/demo/www/index.html

```diff
@@ -21,26 +21,26 @@
                 </a>
             </div>
             <div id="file-picker">
-                <div id="local-file-container">
-                    <span>Local SWF:</span>
-                    <input type="file" accept=".swf,.spl" id="local-file" />
-                </div>
+                <input style="display: none;" type="file" accept=".swf,.spl" id="local-file" />
                 <div id="sample-swfs-container" class="hidden">
```

Comment:
`sample-swfs-container` is no longer needed because it shouldn't be hidden when `swfs.json` doesn't exist.
Be sure to remove it from JS and CSS as well.